applications/api/initialize/init_hertz.go

```diff
@@ -182,7 +185,12 @@ func InitHertz() (*server.Hertz, func()) {
 		}
 	}
 
-	return h, func() {
-		p.Shutdown(context.Background())
+	if ViperConfig.Viper.GetBool("Otel.Enable") {
+		return h, func() {
+		}
+	} else {
```

Comment:
We prefer this :

```
if ... {
    return a, b
}

return c, d
```
instead of

```golang
if ... {
    return a, b
} else {
    return c, d
}
```